### ISSUE TITLE
Hide merge commits from "changes since last review"

### ISSUE THREAD

**Issue by asportnoy** [microsoft/vscode-pull-request-github#4510]
Like with the web view, changes from merge commits shouldn't be shown when viewing changes since the last review.

**Comment by seantleonard** [comment]
I would like to see this too. Whenever changes from `main` are merged into a current working branch, those changes then become noise within the view "Changes since last review." The PR Extension is unusable then. Because I don't want to use "review all changes" because I don't want to keep re-reviewing code changes I've already seen.

**Comment by sgrigorjev** [comment]
Yes, it's a pity that there is no such checkbox, it would be very convenient. I hope this will be added to github someday

**Comment by etsibulko** [comment]
Totally agree, that would be amazing to have such ability.

**Comment by sseficha** [comment]
Agreed! This makes such a cool feature unusable the moment someone merges from main, what a pity it's being overlooked...

**Comment by DamienHagarty** [comment]
I've just started using Github and the VSCode extension is a life saver when it comes to reviews, with the Github web UI seemingly unable to show your review comments along with the changes that have been made to fix them. But this issue is making the extension unusable in some circumstances which is a massive shame. Please look into this.

**Comment by scintillavoy** [comment]
I'm not sure how GitHub generates the condensed diff for merge commits, but the combined diff produced by `git show -c <merge-commit>` or `git show --cc <merge-commit>` appears to match it closely. See the [git show documentation](https://git-scm.com/docs/git-show#Documentation/git-show.txt--c) for details. It would be helpful if we could review only this diff.
